src/app/parcels/edit/editParcelForm.tsx

```diff
@@ -16,14 +18,14 @@ interface EditParcelFormProps {
 }
 
 const prepareParcelDataForForm = (
-    parcelData: ParcelWithCollectionCentre,
+    parcelData: ParcelWithCollectionCentreAndPackingSlot,
     deliveryPrimaryKey: string
 ): ParcelFields => {
     return {
         clientId: parcelData.client_id,
         voucherNumber: parcelData.voucher_number,
-        packingDate: parcelData.packing_datetime,
-        packingTime: parcelData.packing_datetime,
+        packingDate: parcelData.packing_date,
+        packingSlot: parcelData.packing_slot!.primary_key,
```

Comment:
I'd rather not have this force-cast here if `packing_slot` can really be null or undefined (this would crash)